### PR TITLE
Apply new design

### PR DESCRIPTION
**Pull request by TsvetinaIvanova**

### CODE REVIEW

**Comment by didiepetrova** [comment]
:+1:

**Comment by TsvetinaIvanova** [comment]
Added font awesome to project to make it available for WP devices. Added a DisallowOverscroll=true setting.

File: sample-media-player/index.html [hunk 1]
```diff
@@ -1,33 +1,70 @@
...
+			</div>
+			<div class="col -text-left">
+				<button class="button button--rounded" id="buttonStopAudio">
+					<i class="fa fa-stop"></i>
+				</button>
+			</div>
+		</div>
+	</div>
+	<div class="footer">
+		<span class="u-text-right u-color-light">Cordova Plugins Samples</span>
```
Comment (Mike4o) [inline sample-media-player/index.html]: `<span>Feature Examples<span>`

**Comment by Mike4o** [comment]
Suggestion: Add a tip that the sample uses font awesome in the Readme.md file

**Comment by TsvetinaIvanova** [comment]
Applied the two suggestions.

**Comment by Mike4o** [comment]
:+1:

**Pull request merged**